tests/arch/arm/arm_thread_swap/src/arm_thread_arch.c

```diff
@@ -246,6 +253,25 @@ static void alt_thread_entry(void)
 		"CONTROL.FPCA is not cleared at initialization: 0x%x\n",
 		__get_CONTROL());
 
+	/* Verify that the _current_ (alt) thread is
+	 * initialized with mode.FPCA cleared
+	 */
+	zassert_true((_current->arch.mode & CONTROL_FPCA_Msk) == 0,
+		"Alt thread FPCA flag not clear at initialization\n");
+#if defined(CONFIG_MPU_STACK_GUARD)
```

Comment:
Should we add test configuration to force  CONFIG_MPU_STACK_GUARD activation and be sure it is tested ?

Reply:
Oh, but this should be done by default, @ABOSTM, I guess, by default, ztests are enabling HW_STACK_PROTECTION, which gives MPU_STACK_GUARD on ARMv7-M.